app/routes/checkout.jsx

```diff
@@ -83,16 +85,19 @@ function ProductSummary({ product }) {
         }}
       >
         <Text>{product.name}</Text>
-        <Text color="dimmed">{product.category}</Text>
         <Text color="dimmed">{product.code}</Text>
-        <Text color="dimmed">{product.surface}</Text>
-        <Text color="dimmed">{product.size}</Text>
-        <Text color="dimmed">{product.type}</Text>
+        <Text color="dimmed">{product.category + "/" + _.capitalize(product.type) + "/" + _.capitalize(product.surface)}</Text>
```

Comment:
Can you try to find icon and put them to the left of each description? I think this will look better.